pkg/core/native/nativenames/names.go

```diff
@@ -14,3 +14,18 @@ const (
 	CryptoLib   = "CryptoLib"
 	StdLib      = "StdLib"
 )
+
+// Names represents the list of all native contracts
+var Names = []string{
```

Comment:
I see we need to initialize if values are missing. But again, i think it could be hidden inside `GetNativeContractHistory()` (return default if section is empty).

Reply:
I removed global `Names`, but I'd prefer to keep the content of `cfg.NativeUpdateHistories` the same as the result of `GetNativeContractHistory()`. Could you, please, take a look at the current implementation?